api/core/utils.py

```diff
@@ -57,78 +82,86 @@ def create_connection(
             second_logical_interface_id
         ).logical_name,
     )
-    return {
-        "connection_id": wemulate_utils.get_connection_by_name(
-            connection_name
-        ).connection_id,
-        "connection_name": connection_name,
-        "first_logical_interface_id": first_logical_interface_id,
-        "second_logical_interface_id": second_logical_interface_id,
-        **_get_standard_parameter_values(),
-    }
+    return ConnectionComplete(
+        **{
+            "connection_id": wemulate_utils.get_connection_by_name(
+                connection_name
+            ).connection_id,
+            "connection_name": connection_name,
+            "first_logical_interface_id": first_logical_interface_id,
+            "second_logical_interface_id": second_logical_interface_id,
+            "incoming": {},
+            "outgoing": {},
+        }
+    )
```

Comment:
```
ConnectionComplete(
            connection_id=wemulate_utils.get_connection_by_name(
                connection_name
            ).connection_id,
            connection_name=connection_name,
            first_logical_interface_id=first_logical_interface_id,
            second_logical_interface_id=second_logical_interface_id,
            incoming={},
            outgoing={},
        }
    )
````
if you set the default value of incoming and outgoing in the model to a empty dict you don't need it here